deploy-agent/deployd/common/helper.py

```diff
@@ -59,7 +60,7 @@ def get_build_id(filename, env_name):
         return False, None
 
     @staticmethod
-    def get_stale_builds(build_timestamps, num_builds_to_retain=2):
+    def get_stale_builds(build_timestamps, num_builds_to_retain=2) -> Generator:
```

Comment:
why the type is generator?

Reply:
This is a generator function since it calls the `yield` keyword. https://stackoverflow.com/questions/43658999/what-is-the-return-type-hint-of-a-generator-function